### DISCUSSION TITLE
Education Benefits

### DISCUSSION THREAD

**Discussion by GuyBerman** [community/community#139489] [GitHub Education] [answered]
### Hi everyone,

Hi,
Been waiting to get the education benefits
Anyone knows how long it suppose to take?
![צילום מסך 2024-09-23 143512](https://github.com/user-attachments/assets/86bf9780-5090-4dc2-97a8-9120941883e0)

**Comment by davevad93** [comment, marked as answer]
Hi @GuyBerman ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @GuyBerman was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by GuyBerman** [reply]
Thanks for the answer

**Reply by davevad93** [reply]
You're welcome @GuyBerman, have a nice day! 👍
